cloudformation/master-server-substack.cfn.yaml

```diff
@@ -538,7 +538,7 @@ Resources:
                   }
                 - SchedulerSlots: !If
                     - DisableComputeHyperthreading
-                    - !Ref 'ComputeCoreCount'
+                    - cores
```

Comment:
What about SchedulerSlots in compute-fleet-substack? do we need to change that part as well?

Reply:
We don't have to change SchedulerSlots in compute-fleet-substack, because when compute instance update, cfnconfig in compute node will update as well, the number of cfn_scheduler_slots is correct.
Another aspect,
for `node.default['cfncluster']['cfn_instance_slots']`, 
go to this branch
`elsif node['cfncluster']['cfn_scheduler_slots'] == 'cores' node['cpu']['cores']`
or go to this branch 
`else node['cfncluster']['cfn_scheduler_slots']`
will get the same result